releases/stellar/2025-02-GMP-v1.1.0.md

```diff
@@ -0,0 +1,212 @@
+3. Migrate the axelar-operators contract's storage schema key from 'Operators' to 'Operator'.
+4. Add 'stellar/operators.js is-operator', 'stellar/operators.js add-operator', 'stellar/operators.js remove-operator', 'stellar/operators.js collect-fees', 'stellar/operators.js refund', and 'stellar/operators.js execute' commands to interact with the axelar-operators contract.
+
+Deployment changes in the release:
+
+1. Add 'stellar/gateway.js execute' command to execute GMP messages from the CLI.
+2. Make gas-token-address and gas-fee-amount optional for 'stellar/gmp.js send' command.
+3. Add 'stellar/contract.js paused', 'stellar/contract.js pause', 'stellar/contract.js unpause' commands to check the pausable state of a Stellar contract.
+4. Make minimum-rotation-delay optional for 'stellar/deploy-contract.js' commands.
+5. Add 'evm/gateway.js --action isContractCallApproved' command to verify if a gateway contract call is approved from the CLI.
+6. Add 'evm/gateway.js --action isMessageApproved' command to verify if a gateway message is approved from the CLI.
+
+## Deployment
+
+Create an `.env` config. `CHAIN` should be set to `stellar` for mainnet, and `stellar-2025-q1` for all other networks.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAIN=xyz
+```
+
+An initial Stellar chain config needs to be added to `${ENV}.json` file under `chains` key.
+
+### Devnet Amplifier / Stagenet / Testnet
+
+```bash
+\"$CHAIN\": {
+    "name": "Stellar",
+    "axelarId": \"$CHAIN\",
+    "rpc": "https://soroban-testnet.stellar.org",
+    "horizonRpc": "https://horizon-testnet.stellar.org",
+    "networkType": "testnet",
+    "chainType": "stellar",
+    "decimals": 7,
+    "finality": "1",
+    "approxFinalityWaitTime": 1,
+    "tokenSymbol": "XLM",
+    "tokenAddress": "CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC",
+    "explorer": {
+      "name": "Stellar Expert",
+      "url": "https://stellar.expert/explorer/testnet"
+    },
+  "contracts": {}
+  }
+```
+
+### Mainnet
+
+```bash
+"stellar": {
+    "name": "Stellar",
+    "axelarId": "stellar",
+    "rpc": "https://mainnet.sorobanrpc.com",
+    "horizonRpc": "https://horizon.stellar.org",
+    "networkType": "mainnet",
+    "chainType": "stellar",
+    "decimals": 7,
+    "finality": "1",
+    "approxFinalityWaitTime": 1,
+    "tokenSymbol": "XLM",
+    "tokenAddress": "CAS3J7GYLGXMF6TDJBBYYSE3HQ6BBSMLNUQ34T6TZMYMW2EVH34XOWMA",
+    "explorer": {
+      "name": "Stellar Expert",
+      "url": "https://stellar.expert/explorer/public"
+    },
+  "contracts": {}
+  }
+```
+
+1. Request Tokens from Faucet
+
+```bash
+node stellar/faucet.js
+```
+
+2. Retrieve full list of operators used by the relayer.
+
+| Network              | `axelar_operators address`|
+| -------------------- | ---------------------------------------------------------- |
+| **Devnet-amplifier** | `CABRZHKAJVJFX5IZWL7KPA6ZEX6I347W7GBXAUX6WVOGUUBRWCDGOUSI` |
```

Comment:
This is stored in the config, no need to mention here. Only inputs that need to be provided are necessary